server.js

```diff
@@ -64,10 +63,25 @@ app.prepare().then(() => {
       // path /(help|chat) -> Discord chat invite
       res.writeHead(301, { Location: 'https://discordapp.com/invite/dvwXA2N' })
       res.end()
+    } else if (/^\/(docs|documentation)(\/.*)?$/i.test(pathname)) {
+      // path /docs... or /documentation... -> /doc...
+      res.writeHead(301, {
+        Location: req.url.replace(/\/(docs|documentation)/i, '/doc')
+      })
+      res.end()
     } else if (/^\/doc\/commands-reference(\/.*)?/.test(pathname)) {
```

Comment:
`$` is probably missing

Reply:
As in `/^\/doc\/commands-reference(\/.*)?$/`? I don't think it makes a difference. Both would work, yes.

Reply:
Actually you're right, better to include those `$` in regexes to avoid catching longer strings if the `(\/.*)?` part is not present (e.g. `/doc/commands-references`). Done in e76f33bc